package.json

```diff
@@ -40,6 +40,7 @@
     "tls-connect": "^0.2.2"
   },
   "optionalDependencies": {
+    "wrtc": "0.0.59",
```

Comment:
To be clear, "breaks the build" in this case means "npm install dies", not that any of our grunt processes fail. node-wrtc just doesn't compile nicely under OSX - they have related issues filed in their own repo.